src/t8_schemes/t8_standalone/t8_standalone_elements.hxx

```diff
@@ -153,8 +153,9 @@ typedef uint8_t t8_element_level;
 typedef uint8_t t8_cube_id;
 typedef uint8_t t8_child_id;
 
-template <t8_eclass TEclass>
-using t8_element_type = std::bitset<T8_ELEMENT_NUM_EQUATIONS[TEclass]>;
+//template <t8_eclass TEclass>
+//using t8_element_type = std::bitset<T8_ELEMENT_NUM_EQUATIONS[TEclass]>;
+//using t8_element_type = u_int8_t;
```

Comment:
why are you not using this line? It would add a little bit of context to the data-type, especially when it is used as return-type.